README.md

```diff
@@ -1,2 +1,65 @@
-# ros_qualisys
-ROS wrapper around the https://github.com/qualisys/qualisys_cpp_sdk package
+ros_qualisys
+============
+
+This is ROS wrapper around the https://github.com/qualisys/qualisys_cpp_sdk package.
+
+## Getting started
+
+### Dependencies
+
+All dependencies are listed in the [package.xml](https://github.com/Gepetto/ros-qualisys/blob/main/package.xml) file.
+
+For building the package:
+- git
+- doxygen
+- cmake
+
+For executing the package:
+- catkin + roscpp (if with ROS1)
+- ament_cmake + rclcpp (if with ROS2)
+- Python 2 or 3 (Standard Ubuntu 18.04 or higher)
+- boost (Standard Ubuntu 18.04 or higher)
+- qualisys_cpp_sdk (installable via the robotpkg ppa)
+- tf2 (ROS 1 or 2)
+- tf2_ros (ROS 1 or 2)
+- geometry_msgs (ROS 1 or 2)
+
+### Install from source
+
+You can download this package via github:
+
+    git clone --recursive git@github.com:Gepetto/ros-qualisys.git
+
+And then build the package via CMake and Make:
+
+    source /opt/openrobots/setup.bash
+    source /opt/ros/XXXX/setup.bash
+    cd ros-qualisys
+    mkdir _build
+    cd _build
+    cmake .. -DCMAKE_INSTALL_PREFIX=install
```

Comment:
If you document this prefix, you might want to document that the user has to setup this prefix to its environment, otherwise the following `roslaunch` will likely fail.

Also, documenting only the pure cmake version while this is a ROS package will probably mislead users who might be confortable with catkin / colcon, and won't understand why we document something else.

Reply:
fair point I will explain the colcon/catkin way...

Reply:
I this enough as change?